### DISCUSSION TITLE
Does GIT LFS REST API support for bit bucket also

### DISCUSSION THREAD

**Discussion by mantri-mounika** [git-lfs/git-lfs#5885] [Q&A]
Does GIT LFS REST API support for bit bucket also?

**Comment by chrisd8088** [comment]
Thanks for the question!  I think what you're asking if if Atlassian's Bitbucket service supports Git LFS and provides the necessary APIs.  If so, I believe the answer is yes, from their documentation:

- https://support.atlassian.com/bitbucket-cloud/docs/use-git-lfs-with-bitbucket/
- https://support.atlassian.com/bitbucket-cloud/docs/manage-large-files-with-git-large-file-storage-lfs/

If you have any problems setting it up, you'll need to reach out to Atlassian's support team for assistance, as this project is just for the open-source Git LFS client, so we can't help too much with the details of individual Git LFS service implementations.
